src/conftests.py

```diff
@@ -21,8 +21,8 @@
 async_session_maker_test = async_sessionmaker(engine_test, class_=AsyncSession, expire_on_commit=False)
 
 
-@pytest.fixture(scope="function")
-async def prepare_database():
+@pytest.fixture(scope="session")
```

Comment:
it should have scope=function if we name it as session in DB terms

Reply:
looks like you want to have two separate fixture one for setup database when tests running and clean up when all test complete and another fixture to work with database in each test case 
```
def setup_database(): ... - have session scope
def session(setup_database): .... - have function scope
```

Reply:
Moved "setup_database()" fixture into scope=session. Add "session" fixture with scope=function. Removed unnecessary functions and constants. Removed print function